ipaserver/plugins/group.py

```diff
@@ -366,9 +370,16 @@ def pre_callback(self, ldap, dn, *keys, **options):
     def post_callback(self, ldap, dn, *keys, **options):
         assert isinstance(dn, DN)
         try:
+            # A user removing a group may have no rights to remove
+            # an associated policy. Make sure we log an explanation
+            # in the Apache logs for this.
             api.Command['pwpolicy_del'](keys[-1])
-        except errors.NotFound:
-            pass
+        except (errors.NotFound, errors.ACIError) as e:
```

Comment:
I'd prefer if you replace the isinstance check with two separate except blocks:

```
        try:
            ...
        except errors.ACIError as e:
            logger.warning(
                "While removing group %s, user lacked permissions "
                "to remove corresponding password policy. This is "
                "not an issue and can be ignored.", keys[-1]
            )
        except errors.NotFound:
            pass
```

(I also added a full stop to the end of the second sentence.)